src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7511,4 +7523,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_UnsupportedPrimaryConstructorParameterCapturingRefAny" xml:space="preserve">
     <value>Cannot use primary constructor parameter of type '{0}' inside an instance member</value>
   </data>
+  <data name="ERR_BadUsingStaticType" xml:space="preserve">
+    <value>'{0}' type is not valid for 'using static'. Only a class, struct, interface, enum, delegate, or namespace can be used.</value>
```

Comment:
>'{0}' [](http://example.com/codeflow?start=11&length=5)

Not starting a sentence with a capital letter might look strange.

Reply:
Personally, i found the messages clear :)

Reply:
As long as it's a `'`, I think it'll be fine.